File: Chapter 04/Python code/4_11.py
n=int(input("Enter a number: "))
if n==1:
    print("it is neither prime or composite")
    exit(1)
flag =0
for i in range(2,n-1):
    if(n%i==0):
        flag =1
        break
if flag==0:
    print(n, "is a prime number")
else:
    print(n, "is not a prime number")

File: Chapter 04/C Code/4_11.c
#include<stdio.h>
void main()
{
int n, flag,i;
printf("Enter a number: ");
scanf("%d", &n);
if (n==1)
{
printf("it is neither prime or composite");
exit(1);
}
flag =0;
for (i=2; i<n;i++)
{
if(n%i==0)
{
flag =1;
break;
}
}
if (flag==0)
printf( "%d is a prime number", n);
else
printf("%d is not a prime number", n);
}
